river_dl/torch_models.py

```diff
@@ -181,7 +197,14 @@ def nconv(x, A):
     return torch.einsum('ncvl,vw->ncwl', (x, A)).contiguous()
 
 class GraphConvNet(nn.Module):
-    def __init__(self, c_in, c_out, dropout, support_len=3, order=2):
+    def __init__(self, c_in, c_out, dropout, support_len=3, order=2, seed = None):
+
+        if seed:
+            torch.manual_seed(seed)
+            torch.cuda.manual_seed(seed)
+            torch.backends.cudnn.deterministic = True
+            torch.backends.cudnn.benchmark = False
+
```

Comment:
This is only ever wrapped in gwnet, so I think setting the seeds here _and_ in gwnet are unnecessary.
```suggestion
    def __init__(self, c_in, c_out, dropout, support_len=3, order=2):
```